gnomad/resources/grch38/gnomad.py

```diff
@@ -497,6 +500,7 @@ def gnomad_gks(
             coverage_ht = hl.read_table(
                 coverage("genomes").versions[coverage_version].path
             )
+        ht = ht.annotate(mean_cov=coverage_ht[ht.locus].mean)
```

Comment:
```suggestion
        ht = ht.annotate(meanDepth=coverage_ht[ht.locus].mean)
```
only runs if skip-coverage is not set, great implementation. If we are calling it meanDepth later in the code though , would be a good idea to keep it consistent

Reply:
i like the suggestion to keep "cov" and "depth" consistent, but we should keep variables as snake case rather than camel case, how about "mean_cov" -> "mean_depth"?